.env.example

```diff
@@ -24,4 +24,4 @@ MAIL_HOST=mailtrap.io
 MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
-MAIL_ENCRYPTION=null
+MAIL_ENCRYPTION=null
```

Comment:
Not a big deal, but there should be a newline at the end of the file or Git gets angryyy.